WLC/utils/azure.py

```diff
@@ -63,13 +64,40 @@ def save_image_to_azure(self, container, image):
 
         return True, hashed
 
-    def get_tests_from_azure(self, test_key):
-        containers = ['template_code', 'test_cases', 'expected_responses']
+    def save_template_and_test(self, container, template_file, test_file):
+        self.create_container_not_exists(container)
+
+        template = template_file.read()
+        test = test_file.read()
+
+        hashed = hashlib.md5(template).hexdigest()
+
+        template_filename = '{}.py'.format(hashed)
+        test_filename = '{}.json'.format(hashed)
+
+        try:
+            self._block_blob_service.create_blob_from_text(container, template_filename, template)
+            self._block_blob_service.create_blob_from_text(container, test_filename, test)
+        except Exception:
```

Comment:
Why wrap it by an exception? Does it ever throw that?

Reply:
If we doing things with azure and something happens to the network, the api will blow up. It's usually better to catch such errors rather than bringing the backend down.

Reply:
It would not bring the backend down only that request would fail but I agree. I guess we should do the same for other requests.